pro/rac.rst

```diff
@@ -0,0 +1,79 @@
+===================================
+Netmaker Remote Access Client (RAC)
+===================================
+
+The Netmaker Remote Access Client, RAC for short is a GUI tool for easily getting access to a Netmaker network.
+RAC is mostly suited for offsite machines that need access to a Netmaker network. It supports Windows, Mac and Linux (mobile support coming soon).
+
+
+***********************
+Download/Installation
+***********************
+
+You can download the latest version of RAC from the Netmaker file server:
+
+.. code-block::
+
+  https://fileserver.netmaker.org/releases/download/<version>
+
+Search for remote client and download the appropriate version for your operating system.
+
+For Linux (Debian/Ubuntu), you can also use the following command to download the latest version:
+
+.. code-block:: 
+
+   curl -sL 'https://apt.netmaker.org/remote-client/gpg.key' | sudo tee /etc/apt/trusted.gpg.d/remote-client.asc
+   curl -sL 'https://apt.netmaker.org/remote-client/debian.deb.txt' | sudo tee /etc/apt/sources.list.d/remote-client.list
+   sudo apt update
+   sudo apt search remote-client  # to see available versions
+   sudo apt install remote-client
+
```

Comment:
Probably mention that remote-client will be available in the app section, or basically mention how to launch the app

Reply:
addressed